Packs/Zimperium/Integrations/ZimperiumV2/ZimperiumV2.yml

```diff
@@ -0,0 +1,1294 @@
+      description: The branding policy identifier of the Policy Group.
+      type: Unknown
+    - contextPath: Zimperium.PolicyGroup.created
+      description: The date and time the policy group was created.
+      type: Date
+    - contextPath: Zimperium.PolicyGroup.description
+      description: The description of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.dormancyPolicyId
+      description: The dormancy policy identifier of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.emmConnectionId
+      description: The emmConnectionId of the Policy Group.
+      type: Unknown
+    - contextPath: Zimperium.PolicyGroup.emmGroupId
+      description: The emmGroupId of the Policy Group.
+      type: Unknown
+    - contextPath: Zimperium.PolicyGroup.emmPriority
+      description: The emm priority of the Policy Group.
+      type: Unknown
+    - contextPath: Zimperium.PolicyGroup.extensionPolicyId
+      description: The extension policy identifier of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.content.global
+      description: Whether the policy group is global.
+      type: Boolean
+    - contextPath: Zimperium.PolicyGroup.knoxPolicyId
+      description: The KnoxPolicyId of the Policy Group.
+      type: Unknown
+    - contextPath: Zimperium.PolicyGroup.modified
+      description: The date and time when the Policy Group was last modified.
+      type: Date
+    - contextPath: Zimperium.PolicyGroup.name
+      description: The name of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.networkPolicyId
+      description: The NetworkPolicyId of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.osRiskPolicyId
+      description: The OS Risk Policy ID of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.phishingPolicyId
+      description: The phishing policy identifier of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.privacyId
+      description: The privacy identifier of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.team.id
+      description: The ID of the team associated with the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.team.name
+      description: The Team name of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.trmId
+      description: The trmId of the Policy Group.
+      type: String
+    - contextPath: Zimperium.PolicyGroup.team
+      description: The Policy Group's team information.
+      type: Unknown
+  - arguments:
+    - description: The identifier of the policy. Can be retrieved using zimperium-policy-group-list in the Zimperium.PolicyGroup.privacyId field.
+      name: policy_id
+      required: true
+    description: Get privacy policy by its identifier.
+    name: zimperium-policy-privacy-get
```

Comment:
policy privacy or privacy policy? make sure to be coherent (also in the outputs and the code)

Reply:
It is a privacy policy. 
But command name and outputs by the design docs are:
zimperium-policy-privacy-get, PolicyPrivacy.